packages/ui/src/utils/channelUtils.tsx

```diff
@@ -8,8 +8,8 @@ export function getChannelTitle(channel: db.Channel) {
   if (channel.type === 'dm') {
     const member = channel.members?.[0];
     if (!member) {
-      console.warn('bad dm channel', channel.id, 'missing contact');
-      return 'Untitled DM';
+      console.log('bad dm channel', channel.id, 'missing contact');
```

Comment:
It was triggering a bunch before I fixed contact passing. It will still happen if we get invited and don't have the contact already. Backend should automatically add it for DMs, but I need to look again at if/how we're subbing to that from the client.

In any case, I left it bumped down to a log vs warning since we have a meaningful fallback with the @p.